frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -581,6 +587,7 @@ export type DiskModalProps = {
   showInitialValidation?: boolean;
   isCreateTemplate?: boolean;
   isEditing?: boolean;
+  isWindows?: boolean;
```

Comment:
please, let's not add more properties to this component. We can pass the desired value inside the disk value in place where we need it.